10.0/alpine-fpm/Dockerfile

```diff
@@ -0,0 +1,67 @@
+FROM php:7.0-fpm-alpine
```

Comment:
We should use the same php version as used by the other images, this would mean for nextcloud 9/10 php 5.6 and for nextcloud 11 php 7.1.